COMDOX-1722: Ai docs section (#933)
commerce-docs/microsite-commerce-storefront: src/content/docs/ai/ai-agent-skills.mdx

After the install completes, restart your coding agent so it picks up the new skills and MCP configuration.

> Different agents and IDEs expose skills in different ways, so there is no single way to confirm that the skills are installed correctly. Check your agent's or IDE's documentation for how to verify that it has loaded your project's skills. For example, in Cursor you can open the configuration or settings area to see the list of skills it has detected. Other agents use different UIs or indicators.

Reviewer: ```suggestion
Different agents and IDEs expose skills differently, so there is no single way to confirm installation. See your agent or IDE documentation for how to verify that your project’s skills are loaded. For example, in Cursor, you can open the settings or configuration panel to view detected skills. Other agents provide their own interfaces or indicators for this purpose.
```

- **For drop-in customizations**, the agent uses the researcher skill to look up the correct slot names, event payloads, and API functions from the actual source before writing code — not from assumptions.
- **For block development**, the agent follows boilerplate conventions: DOM manipulation with `document.createElement()`, CSS selectors scoped to the block name, and checking `/blocks` for existing solutions before creating new ones.
- **For content structure**, the agent uses the content modeler skill to design block tables that work for both developers and content authors.
- **After implementation**, the agent uses the tester skill to verify the result in a real browser, not by reviewing code.

**Example prompt** — You can trigger the planning workflow and developer skills with a single, detailed request. For example:

> > Use the planning workflow to add a button to the product pages that allows the user to share the product quickly on social media platforms like Facebook and Twitter. The button should be displayed below the title section on the product detail page.

Reviewer: ```suggestion
> Use the planning workflow to add a social sharing button to product pages. Place the button below the product title on the product detail page, allowing users to quickly share the product on platforms such as Facebook and Twitter.
```
